src/Domain/Fund.php

```diff
@@ -27,28 +21,26 @@
  */
 #[ORM\Table]
 #[ORM\Entity(repositoryClass: FundRepository::class)]
-#[ORM\InheritanceType('SINGLE_TABLE')]
-#[ORM\DiscriminatorColumn(name: 'fundType', type: 'string')]
-#[ORM\DiscriminatorMap([
-    'unknownFund' => self::class,
-    ...DISCRIMINATOR_MAP,
-])]
 #[ORM\HasLifecycleCallbacks]
-abstract class Fund extends SalesforceReadProxy
+class Fund extends SalesforceReadProxy
 {
-    use TimestampsTrait;
-
-    /** @var positive-int */
-    public const int NORMAL_ALLOCATION_ORDER = 999;
```

Comment:
the 999 I think was never used for anything since it's overidden by all child classes.

Reply:
Yeah, should have just been abstract probably as the class can't be used as-is.

Reply:
I don't think you can have an abstract const, but it could be an abstract method. In PHP 8.4 it could be an abstract property.